lms/static/js/spec/main.js

```diff
@@ -762,6 +767,7 @@
         'js/spec/learner_dashboard/program_card_view_spec.js',
         'js/spec/learner_dashboard/certificate_view_spec.js',
         'js/spec/api_admin/catalog_preview_spec.js',
+        'js/spec/commerce/receipt_spec.js'
```

Comment:
I wonder why we have to enumerate all of these files one-at-a-time. @ehteshamkafeel do you remember? It makes this very painstaking.

If this list is necessary, I'd like to see it be in alphabetical order so that we can find whether a file is included or not. The current semi-random order isn't helpful.